src/bridge/cockpitpcpmetrics.c

```diff
@@ -112,16 +121,23 @@ build_meta (CockpitPcpMetrics *self,
   JsonArray *instances;
   JsonObject *root;
   pmValueSet *vs;
-  gint64 timestamp;
+  struct timeval now_timeval;
+  gint64 timestamp, now;
   char *instance;
   int i, j;
   int rc;
 
+  gettimeofday (&now_timeval, NULL);
```

Comment:
The metrics are using monotonic time (see CockpitMetrics). Is it intentional that we use wall time here? Just double checking.